src/core/stdc/stdio.d

```diff
@@ -1373,7 +1407,9 @@ else version (CRuntime_Microsoft)
     ///
     pure int  ferror(FILE* stream);
     ///
-    pure int  fileno(FILE* stream);
+    pure int  _fileno(FILE* stream);
+    ///
+    alias fileno = _fileno;
```

Comment:
This is also already taken care of by https://github.com/dlang/druntime/blob/27e0bb8b957675ef6af91a86a1af7f9a3f449b9c/src/rt/msvc.d#L201-L204 for older MSVC runtimes.

Reply:
The alias is more for the benefit of `std.stdio` than anything else.
https://github.com/dlang/phobos/blob/e9014c43428ac3f01d02fe28b2845074e7c80421/std/stdio.d#L207-L222

Reply:
Yeah but resorting to the MS-specific name wasn't needed before either; newer MS runtimes have that symbol which was previously just declared, and in case the MS runtime doesn't have it, there's the fallback in the other module (and could be added to your gdc module as well). So Phobos could just use `core.stdc.stdio.fileno` for `CRuntime_Microsoft` today.